CHANGES/1648.feature

```diff
@@ -0,0 +1,3 @@
+Added support for the Replication feature. The replication process allows a Pulp instance to
```

Comment:
Can you mention that on the replicated Pulp there will be created only read-only repos? i.e. it won't be possible to push content into them.

Could you add a mention on replication support in the docs?